packages/@react-aria/overlays/src/useOverlay.ts

```diff
@@ -66,9 +70,12 @@ export function useOverlay(props: AriaOverlayProps, ref: RefObject<Element>): Ov
     isOpen,
     isDismissable = false,
     isKeyboardDismissDisabled = false,
-    shouldCloseOnInteractOutside
+    shouldCloseOnInteractOutside,
+    preventScroll = true
   } = props;
 
+  usePreventScroll({isDisabled: !preventScroll || !isOpen});
```

Comment:
I don't think we should change the default behavior of useOverlay, it's probably a breaking change. I think it's fine to call `usePreventScroll` from inside `Popover` though, just like we do `Modal`. React Aria users can do the same.

Maybe in the new overlay hooks I'm working on in #3347 we can enable this by default (since those are opt-in).

Reply:
Reverted the change. I like the idea of enabling it in the new hooks.